beacon-chain/blockchain/new_slot.go

```diff
@@ -36,6 +36,10 @@ func (s *Service) NewSlot(ctx context.Context, slot types.Slot) error {
 		return errors.Wrap(err, "could not reset boosted proposer root in fork choice")
 	}
 
+	// Insert slashings to forkchoice.
+	slashings := s.cfg.SlashingPool.PendingAttesterSlashings(ctx, s.head.state, true)
+	s.insertSlashingsToForkChoiceStore(ctx, slashings)
```

Comment:
What worries me about this approach is that we will be going over the following loops on every slot:
1. Check that each slashing has not been included on chain or not
2. Send the slashings to forkchoice, which will be a noop (I will fix this now) 
We shouldn' t be sending all slashings every single slot until the pending queue dissapears.